### ISSUE TITLE
FormCaption & FormCaptionGroup - fix accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#779]
#### Description / User Story

The following errors and incomplete tests are thrown by the storybook accessibility addon and should be fixed:
- FormCaption: Incomplete: Elements must only use permitted ARIA attributes (if ariaLabel has a value)
- FormCaptionGroup: Incomplete: Elements must only use permitted ARIA attributes (probably inherited from FormCaption if ariaLabel has a value)

----
#### Requirements
- [x] #544 

----
#### Acceptance Criteria
- [x] All errors and incomplete tests thrown by the accessibility addon are resolved for the FormCaption:
   - [x] Incomplete: Elements must only use permitted ARIA attributes (if ariaLabel has a value)
- [x] All errors and incomplete tests thrown by the accessibility addon are resolved for the FormCaptionGroup:
   - [x] Incomplete: Elements must only use permitted ARIA attributes (probably inherited from FormCaption if ariaLabel has a value)

----
#### Background information
- ...
